java-frontend/src/main/java/org/sonar/java/se/checks/debug/DebugMethodYieldsOnInvocationsCheck.java

```diff
@@ -50,23 +52,20 @@
   tags = "debug")
 public class DebugMethodYieldsOnInvocationsCheck extends SECheck implements DebugCheck {
 
-  private Map<IdentifierTree, MethodBehavior> methodBehaviors = new HashMap<>();
+  private Deque<Map<IdentifierTree, Symbol.MethodSymbol>> methodSymbols = new LinkedList<>();
```

Comment:
I think we don't really need a map between identifiers and method symbols, we can just keep list of method invocations and get symbols in `reportAll` method